libavpipe/src/avpipe_xc.c

```diff
@@ -886,20 +884,8 @@ set_nvidia_params(
     if (params->gpu_index >= 0)
         av_opt_set_int(encoder_codec_context->priv_data, "gpu", params->gpu_index, 0);
 
-    /*
-     * The encoder_codec_context->profile is set just for proper log message, otherwise it has no impact
-     * when the encoder is nvidia.
-     */
-    int profile = avpipe_nvh264_profile(params->profile);
-    if (profile > 0) {
-        av_opt_set_int(encoder_codec_context->priv_data, "profile", profile, 0);
-        encoder_codec_context->profile = profile;
```

Comment:
This setting of the profile variable is not doing anything (and there is a comment above about it not doing anything) - I decided to clean both up